frontend-app/src/TemperatureControl.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+function TemperatureControl() {
+    const [count, setTemp] = useState(10);
+    const maxTemp = 30;
+    const minTemp = 0;
+       
+const tempUp = () => {
+    if (count < maxTemp)
+     setTemp(count + 1);
+    
+}
+
+const tempDown = () => {
+    if (count > minTemp)
+      setTemp(count - 1);
+}
+
+if (count >= 15)
+return ( 
+<div class="app-container">
```

Comment:
class лучше не использовать (да так работает, но кучу ошибок сыпать будет)
в остальных местах также нужно исправить
```suggestion
<div className="app-container">
```